src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/AssemblyPart.cs

```diff
@@ -41,6 +51,32 @@ public AssemblyPart(Assembly assembly)
         /// <inheritdoc />
         public IEnumerable<TypeInfo> Types => Assembly.DefinedTypes;
 
+        public IEnumerable<PrecompiledViewInfo> PrecompiledViews
```

Comment:
We should figure out if we're moving view abstractions into MVC.Abstractions

At the **least** this change will unblock you for now.